cmd/types.go

```diff
@@ -105,6 +105,23 @@ type Metadata = map[string]string
 // UnsupportedType is a type alias for v1 version of the k8s apps API, this is meant for testing
 type UnsupportedType = apiv1.Binding
 
+// Resource types listed here
```

Comment:
`// ResourceTypes is a map of all the Kubernetes workloads kubeaudit can decode`

export comments should start with the variable name, in this case `ResourceTypes`